CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes for each version of this project will be documented in this
 
 ## 10.1.0
 
+### General
+- `igxCombo`
+    - **Behavioral Change** - Change default positioning strategy from `ConnectedPositioningStrategy` to `AutoPositionStrategy`. The Auto strategy will initially try to show the element like the Connected strategy does. If the element goes out of the viewport Auto will flip the starting point and the direction, i.e. if the direction is 'bottom', it will switch it to 'top' and so on.
```

Comment:
We should add something like this:

Additionally if after flipping the direction content goes out of the view auto strategy will revert to initial start point and direction and will push the content into the view. Note after pushing the content it may hide the combo's input.